lib/src/exceptions/conditionals/signal_redriven_exception.dart

```diff
@@ -0,0 +1,28 @@
+/// Copyright (C) 2022 Intel Corporation
+/// SPDX-License-Identifier: BSD-3-Clause
+///
+/// signal_redriven_exception.dart
+/// An exception that is thrown when a signal is
+/// redriven multiple times.
+///
+/// 2022 November 9
+/// Author: Yao Jing Quek <yao.jing.quek@intel.com>
+///
+
+import 'package:rohd/rohd.dart';
+
+/// Throws [SignalRedrivenException] whenever a [Logic] signal
+/// is redriven for mutiple times in [Conditional].
+///
+class SignalRedrivenException implements Exception {
+  late final String _message;
+
+  /// constructor for SignalRedrivenException,
```

Comment:
Start sentences with a capital letter, end with a period, in general try to be careful to have good grammar since this is documentation.

https://dart.dev/guides/language/effective-dart/documentation

Reply:
Alright, noted.

Reply:
### References:
Below are references for documentation guidelines. 

```
[1] https://github.com/flutter/flutter/blob/74cfc3db67dc89d617ebccd2763301777106c7fd/packages/flutter_driver/lib/src/common/wait.dart
[2] https://github.com/flutter/flutter/blob/19eab105a57e519753af9556b045eb1b0ccae9fd/packages/flutter_tools/lib/src/build_system/exceptions.dart#L11
[3] https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs
```